crates/uv/src/commands/tool/install.rs

```diff
@@ -276,19 +276,27 @@ pub(crate) async fn install(
         installed_tools
             .get_environment(&from.name, &cache)?
             .filter(|environment| {
-                python_request.as_ref().map_or(true, |python_request| {
-                    if python_request.satisfied(environment.interpreter(), &cache) {
-                        debug!("Found existing environment for `{from}`", from = from.name.cyan());
-                        true
-                    } else {
-                        let _ = writeln!(
-                            printer.stderr(),
-                            "Existing environment for `{from}` does not satisfy the requested Python interpreter",
-                            from = from.name.cyan(),
-                        );
-                        false
-                    }
-                })
+                // NOTE(lucab): this compares `base_prefix` paths as a proxy for
+                // detecting interpreters mismatches. Directly comparing interpreters
+                // (by paths or binaries on-disk) would result in several false
+                // positives on Windows due to file-copying and shims.
+                let old_base_prefix = environment.interpreter().sys_base_prefix();
+                let selected_base_prefix = interpreter.sys_base_prefix();
+                if old_base_prefix == selected_base_prefix {
+                    debug!(
+                        "Found existing interpreter for tool `{}`: {}",
+                        from.name,
+                        environment.interpreter().sys_executable().display()
+                    );
+                    true
+                } else {
+                    let _ = writeln!(
+                        printer.stderr(),
+                        "Ignored existing environment for `{from}` due to stale Python interpreter",
```

Comment:
My intuition is that this should be "Ignoring" but I'm not sure if that's clear in our style guide. If you search though, we use "Ignoring" consistently but not "Ignored".

Is the meaning of "stale" clear here? I worry it's not clear to the user if that means something is wrong with the existing interpreter or not. We might want to say something like "Ignoring existing environment for `{from}`: the requested Python interpreter does not match the environment interpreter". Wdyt?